src/Utils.cpp

```diff
@@ -553,3 +553,77 @@ bool Utils::SetFilenameFormatting(const char* filenameFormatting) {
 	config_save(profile);
 	return true;
 }
+
+obs_data_t* Utils::GetSceneItemPropertiesData(obs_sceneitem_t* sceneItem) {
+	OBSDataAutoRelease data = obs_data_create();
```

Comment:
It's a bit risky to do this. When leaving the current scope, `data` will be automatically released and destroyed because its reference count was `1`. `OBSDataAutoRelease` only performs releases and doesn't add new references.

```suggestion
	obs_data_t* data = obs_data_create();
```